docs/howto/grape.md

```diff
@@ -0,0 +1,13 @@
+# Integration with Grape
+
+To add Grape support, `require 'grape/active_model_serializers'` in the base of your Grape endpoints like so:
```

Comment:
Let's link to `Grape` repo, and maybe mention supported versions of grape are specified in the gemspec.

I think it should also reference the configuration how-to